tests/unit/package_managers/yarn_classic/test_resolver.py

```diff
@@ -179,7 +179,10 @@ def test__is_from_npm_registry_can_parse_incorrect_registry_urls() -> None:
 def test_create_package_from_pyarn_package(
     pyarn_package: PYarnPackage, expected_package: YarnClassicPackage, rooted_tmp_path: RootedPath
 ) -> None:
-    package_factory = _YarnClassicPackageFactory(rooted_tmp_path)
+    package_factory = _YarnClassicPackageFactory(
```

Comment:
We might want a test case where dev=True in the resulting package

Reply:
added